docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -770,6 +770,101 @@ helm upgrade mycluster hpcc/hpcc -f threethorsonequeue.yaml
         </listitem>
       </orderedlist>
     </sect2>
+
+    <sect2 id="configwithlzonly" role="nobrk">
+      <title>Create a Custom Configuration Chart for a Landing Zone
+      only</title>
+
+      <para>In this section, we will create a custom configuration that
+      deploys a "platform" containing only a Landing Zone. This can be useful
+      if all you need is a landing zone server with dafilesrv running. </para>
+
+      <para><emphasis role="bold">Note:</emphasis> This can only be deployed
+      to a different namespace than any other platform instance. </para>
```

Comment:
This is generally true. You can't deploy >1 hpcc chart to the same namespace. Maybe this comment should be moved and generalized.